tiup/tiup-documentation-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+title: TiUP Documentation Guide
+category: tools
+---
+
+# TiUP Documentation Guide
+
+## TiUP user guide
+
+- [TiUP Overview](/tiup/tiup-overview.md): Give an overall introduction to TiUP, for example, how to install and use TiUP
+- [TiUP Terminology and Concepts](/tiup/tiup-terminology-and-concepts.md): Explain the terms that you might bump into when using TiUP, and help you understand the key concepts of TiUP
+- [TiUP Component Management](/tiup/tiup-component-management.md): Introduce all TiUP commands in detail, and how to use TiUP to download, update and delete components
+- [TiUP FAQ](/tiup/tiup-faq.md): Introduce common issues when you use TiUP, including FAQs of the third-party components of TiUP
+- [TiUP Troubleshooting Guide](/tiup/tiup-troubleshooting-guide.md): Introduce the troubleshooting methods and solutions if you encounter issues when using TiUP
+
+## TiUP component user guide
+
+- [tiup-playground](/tiup/tiup-playground.md): Introduce how to use the TiUP playground component to quickly build a local TiDB cluster for testing
+- [tiup-cluster](/tiup/tiup-cluster.md): Introduce the usage and command-line flags of the TiUP cluster component, which is used to deploy and maintain a TiDB cluster for production
+- [tiup-mirror](/tiup/tiup-mirror.md): Introduce how to use the TiUP mirror component to customize a local offline mirror
```

Comment:
TiUP mirror is not a component anymore. It's a subcommand.
At previous it's a component, we use it as below:
```
tiup install mirrors
tiup mirrors clone
```
At this time,  the mirrors component is deprecated and there is a `mirror` sub command:
```
tiup mirror clone xxx // no need to install
```

Reply:
The original Chinese text needs updates too. Can you file a PR to change the Chinese descriptions? I will align your changes later in the English version. @lucklove

Reply:
Confirmed with @lucklove, he will fix the deprecated content later in a new PR.